src/bundle/Resources/config/fieldtype_services.yaml

```
@@ -139,6 +139,12 @@ services:
         tags:
             - {name: ezrichtext.converter.input.xhtml5, priority: 10}
 
+    # Note: should run after xsl transformation
+    ezrichtext.converter.input.xhtml5.listlinebreaks:
```

Can we just use FQCN as service name?
```suggestion
    Ibexa\FieldTypeRichText\RichText\Converter\ListLineBreaks:
```

---

In that case, remove the `class:` entry as well

---

I used the old-style naming in order to be consistent with existing code. Will change to FQCN and remove class then

---

Fixed in 295b2127692c3b2b2aaa26c0c9310cb3c916a2f4